components/LetterTile.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+
+export default class LetterTile extends Component {
+  render() {
+    return (
+      <div>
+        <input type="text">
+
+      </input>
```

Comment:
Be careful with the styling here, you should have a consistent number of spaces at different indentation levels. You can use the included prettier formatter that comes with the repo to automatically fix these style issues (there is also a VSCode extension for Prettier).